pepper-apis/dsm-core/src/test/java/org/broadinstitute/dsm/util/NotificationUtilTest.java

```diff
@@ -0,0 +1,21 @@
+package org.broadinstitute.dsm.util;
+
+import static org.junit.Assert.fail;
+
+import com.typesafe.config.Config;
+import com.typesafe.config.ConfigFactory;
+import org.junit.Test;
+
+public class NotificationUtilTest {
+
+    @Test
+    public void testStartup() {
+        Config cfg = ConfigFactory.parseResources("NotificationUtilTest.conf");
+        try {
+            var notificationUtil = new NotificationUtil(cfg);
+            notificationUtil.startup(cfg);
```

Comment:
Is this actually "starting" anything? The code looks like mostly initialization? 
I ask because this can be confusing when evaluating test code, since presumably if a test starts something -- like a thread -- it needs to get stopped.

Reply:
Agreed, it's pretty confusingly named.  Switching to `initialize` since there's no threading going on here.